generators/neslib/neslib-system-famitracker.template.asm

```diff
@@ -101,6 +101,11 @@ RLE_BYTE    =TEMP+3
 .segment "CODE"
 
 initialize_library:
+    txa
+@1:
```

Comment:
I believe these two lines are leftover code, which don't actually have any impact. To explain, the `@1` label this creates is not used anywhere, and the accumulator set by `txa` is immediately overwritten by the `lda` two lines down.